frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -871,14 +871,14 @@ global.user_groups.add(backend);
     assert_typeahead_equals("hi emoji :ta", emoji_list);
     assert_typeahead_equals("hi emoji :da", emoji_list);
     assert_typeahead_equals("hi emoji :da_", emoji_list);
-    assert_typeahead_equals("hi emoji :da ", emoji_list);
+    assert_typeahead_equals("hi emoji :da ", false);
     assert_typeahead_equals("hi emoji\n:da", emoji_list);
     assert_typeahead_equals("hi emoji\n :ra", emoji_list);
     assert_typeahead_equals(":+", emoji_list);
     assert_typeahead_equals(":la", emoji_list);
     assert_typeahead_equals(" :lee", emoji_list);
-    assert_typeahead_equals("hi :see no", emoji_list);
-    assert_typeahead_equals("hi :japanese post of", emoji_list);
+    assert_typeahead_equals("hi :see no", false);
+    assert_typeahead_equals("hi :japanese post of", false);
```

Comment:
These two cases should still autocomplete.

Reply:
These are returning false cause there is space after `:see` and `:japanese`. When `:see` will be typed in compose box it will return emoji list, not return after typing `:see no` in compose box. I have update the tests, to make it more clear.